src/js/entities-service/actions.js

```diff
@@ -18,8 +18,8 @@ const Entity = BaseEntity.extend({
     ].join();
     return this.fetchModel(id, { data: { include } });
   },
-  fetchActions({ filter }) {
-    const data = { filter };
+  fetchActions({ filter, include }) {
+    const data = { filter, include };
```

Comment:
What is the purpose of the `include` here? Asking out of curiosity.

Reply:
Adding `include` is required for (eg) `?include=patient.patient-fields.scoring` to get the necessary data (value `ranking`) for Risk Sort.

Reply:
https://roundingwell.slack.com/archives/C02U318AW/p1670864213694099

Reply:
It was a deprecated API on the endpoint, but it was added back so we can pass `patient.patient-field.foo`

Reply:
Gotcha, thanks @paulfalgout @shadowhand